.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+Pact Pull Request
+---
+
+Please include the following checklist in your PR:
+
+* [ ] Test coverage for the proposed changes
+* [ ] Documentation has been updated if new natives or FV properties have been added. To generate new documentation, issue `cabal run tests`. If they pass locally, docs are generated.
+* [ ] Any changes that could be relevant to users [have been recorded in the changelog](https://github.com/kadena-io/pact/blob/master/CHANGELOG.md)
+
+Additionally, please justify why you should or should not do the following:
+
+* [ ] Confirm replay/back compat
+* [ ] Benchmark regressions.
```

Comment:
```suggestion
* [ ] Benchmark regressions
* [ ] (For Kadena engineers) Run integration-tests against a Chainweb built with this version of Pact
```

Do you think this is too much? For some changes, I'd really like to see this become a habit.

Reply:
Integration tests usually come with a chainweb version to integrate with, which might be a stretch for pact-only PRs

Reply:
But for Pact engineers, I think we should all get used to building our latest Pact against the latest Chainweb and running a regression on that pair, for any significant changes.

Reply:
I think the hard part here is only experienced engineers can decide if benchmarks, replay or chainweb regression is indicated. So perhaps these should go into the checklist _to be decided on by the reviewer_ who can escalate if they don't know the answer.

Reply:
I like this, the list can be a reminder to the reviewer for things should or *might* want to ask for.